tests/test_spectral_indices.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import Mock
+from openeo.extra.spectral_indices.spectral_indices import _callback, _get_expression_map
+from processes import array_create
+
+x = array_create([9,4,6,1,3,8,2,5,7,11,7,2])
```

Comment:
you should not define global variables like this for tests, you should recreate your input for each test separately